### PR TITLE
clarify -p as per feedback

### PR DESCRIPTION
**Pull request by bcoca**
some people seem to assume -p will add whatever directory to the configured collection paths.

##### ISSUE TYPE

- Bugfix Pull Request
- Docs Pull Request

##### COMPONENT NAME

ansible-galaxy

### CODE REVIEW

File: lib/ansible/cli/galaxy.py [hunk 1]
```diff
@@ -119,7 +119,9 @@ def init_parser(self):
                                      help='The collection(s) name or path/url to a tar.gz collection artifact. This '
                                           'is mutually exclusive with --requirements-file.')
         cinstall_parser.add_argument('-p', '--collections-path', dest='collections_path', required=True,
-                                     help='The path to the directory containing your collections.')
+                                     help='The path to the directory containing your collections. This does not add '
+                                          'it to the configuration, you should use one of the configured paths or '
```
Comment (samccann) [inline lib/ansible/cli/galaxy.py]:
```suggestion
                                          'Use one of the configured paths or '
```

File: lib/ansible/cli/galaxy.py [hunk 2]
```diff
@@ -119,7 +119,9 @@ def init_parser(self):
                                      help='The collection(s) name or path/url to a tar.gz collection artifact. This '
                                           'is mutually exclusive with --requirements-file.')
         cinstall_parser.add_argument('-p', '--collections-path', dest='collections_path', required=True,
-                                     help='The path to the directory containing your collections.')
+                                     help='The path to the directory containing your collections. This does not add '
```
Comment (samccann) [inline lib/ansible/cli/galaxy.py]:
```suggestion
                                     help='The path to the directory containing your collections. '
```

File: lib/ansible/cli/galaxy.py [hunk 3]
```diff
@@ -119,7 +119,9 @@ def init_parser(self):
...
+                                          'it to the configuration, you should use one of the configured paths or '
+                                          'the directory containing the playbook that references the collection.')
```
Comment (samccann) [inline lib/ansible/cli/galaxy.py]:
```suggestion
                                          'the directory containing the playbook that references the collection.'
                                          'Note: this does not add the path to the configured paths.')
```

**Comment by samccann** [comment]
Don't know how well my suggested edits are showing up but the goal was to get how to use the -p option first, and then a final note that it's not adding a path to the configured paths etc.

**Comment by bcoca** [comment]
can you do single edit? these seem to be in conflict to each other.

**Comment by samccann** [comment]
ok created PR off your branch - https://github.com/bcoca/ansible/pull/9  -

**Comment by jborean93** [comment]
Just an FYI, when it goes to install a collection in a path that not's part of `COLLECTIONS_PATH`, it will display the warning https://github.com/ansible/ansible/blob/devel/lib/ansible/cli/galaxy.py#L610.

**Review by jborean93 — APPROVED** [review]

**Comment by bcoca** [comment]
@jborean93 I saw that, my initial intent was to add that warning and then was surprised it already existed (probably ignored by reporter), that said, it still seemed prudent to clarify on the help for the option.

**Pull request closed without merging**